src/components/GIFComponent/GIFComponentStyles.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+const GIFComponentStyled = styled.div`
+  width: ${props => props.width}px;
+  min-height: ${props => props.height}px;
+  padding-bottom: 10px;
+  cursor: pointer;
+  text-align: center;
+  transition: all 0.2s ease-in-out;
+  &:hover {
+    -webkit-transform: scale(1.05, 1.07);
```

Comment:
you don't need this autoprefixes. Styled Components autoprefixes the css automatically